### DISCUSSION TITLE
Allow moosh to configure "External tools"?

### DISCUSSION THREAD

**Discussion by willend** [tmuras/moosh#350] [General]
Hi there,

I have deployed moosh behind the scenes at e-neutrons.org and pan-learning.org - it is very useful for various automation etc.

Would you consider adding functionality for auto-configuration of (e.g. LTI-based) external tools?

Best
Peter

**Comment by klebann** [comment]
I added yesterday config-plugin-import and export function, maybe this will help you?

**Reply by willend** [reply]
Sounds excellent, I will have a look!
